lib/fpm/package/virtualenv.rb

```diff
@@ -0,0 +1,217 @@
+require "fpm/package"
+require "fpm/util"
+require "backports"
+require "fileutils"
+require "find"
+require "socket"
+
+# A virtualenv package.
+class FPM::Package::VirtualEnv < FPM::Package
+    option "--installation-path", "DIRECTORY", "The directory to install the final environment."
```

Comment:
I think we can just reuse fpm's `--prefix` flag here, instead of adding a new flag. Thoughts?